docs/docs/installation/configuring-superset.mdx

```diff
@@ -285,7 +285,7 @@ If you want to rotate the SECRET_KEY(change the existing secret key), follow the
 Add the new SECRET_KEY and PREVIOUS_SECRET_KEY to `superset_config.py`:
 
 ```python
-PREVIOUS_SECRET_KEY = 'CURRENT_SECRET_KEY' # The default SECRET_KEY for deployment is '21thisismyscretkey12eyyh'
-SECRET_KEY = 'YOUR_OWN_RANDOM_GENERATED_SECRET_KEY'
+PREVIOUS_SECRET_KEY = 'CURRENT_SECRET_KEY' # The default SECRET_KEY for deployment is mentioned in superset/constants.py as CHANGE_ME_SECRET_KEY variable
```

Comment:
The default secret key has changed a few times. Might be worth mentioning how they can find it via their previous deployment. Something like 

```shell
$ superset shell
>>> from flask import current_app; print(current_app.config["SECRET_KEY"])
```

Reply:
> The default secret key has changed a few times. Might be worth mentioning how they can find it via their previous deployment. Something like
> 
> ```shell
> $ superset shell
> >>> from flask import current_app; print(current_app.config["SECRET_KEY"])
> ```

I think that's a good Idea. Thanks for your suggestion @nytai